DashWallet/Sources/Models/CrowdNode/CrowdNode+UserDefaults.swift

```diff
@@ -79,6 +81,17 @@ extension CrowdNode {
         get { UserDefaults.standard.bool(forKey: kConfirmationDialogShown) }
         set(value) { UserDefaults.standard.set(value, forKey: kConfirmationDialogShown) }
     }
+    
+    var onlineInfoShown: Bool {
```

Comment:
@Syn-McJ All these 'user defaults' properties are being used for quite often and every time we access the property, we read it from the file system, which is quite slow. I suggest to cache these properties by reading from user default once at initialization stage and when we want to update these properties.

Reply:
Sounds good

Reply:
Fixed

Reply:
Looks good!